src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,30 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty lines before the class declaration to adhere to best practices.